app/webpack/stylesheets/admin.scss

```diff
@@ -1,29 +1,27 @@
 .admin-page {
-  nav.app-navigation {
+  .app-navigation {
     @media (min-width: 40.0625em) {
```

Comment:
Clean as you code (CAYC): we could use the design system's media queries throughout our stylesheets.

Reply:
To be actioned by https://dsdmoj.atlassian.net/browse/AP-3699